src/pudl/analysis/record_linkage/__init__.py

```diff
@@ -2,6 +2,8 @@
 from . import (
     classify_plants_ferc1,
     eia_ferc1_record_linkage,
+    eia_ferc1_record_linkage_splink_model,
+    eia_ferc1_splink_model_config,
```

Comment:
These module names are quite long. Is the idea that their contents will get rolled into the more generic `eia_ferc1_record_linkage` module once splink is the only option? If not, can we simplify them? The fact that they're already in the `record_linkage` subpackage and that `splink` is dedicated to record linkage makes that bit of the name seem like it might be superfluous.  Maybe `eia_ferc1_splink_model` and `eia_ferc1_splink_config`?

Reply:
Yep, the idea is that it would fully replace the other module, in which case I think `eia_ferc1_model` would be fine.